### PR TITLE
Help page init, Problem component init.

### PR DESCRIPTION
**Pull request by Edgar000**

### CODE REVIEW

File: client/containers/problems_database.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        rating: '4',
+        answersCount: '18',
+        isAnswered: true
+    },
+    {
+        id: '3',
+        userName: 'Georg Marvin',
+        creationDate: '3.03.2017',
+        subject: 'javascript',
+        description: 'How can I ng-repeat in React?',
```
Comment (ShevchukJulia) [inline client/containers/problems_database.js]: You already did that using simple items.map(item => \<ItemComponent item={item}/>)

File: client/containers/App.js [hunk 2]
```diff
@@ -5,15 +5,17 @@ import { Footer } from '../components/Footer';
...
 export class App extends Component {
     render() {
         return (
             <div>
                 <Header />
                 <Router history={browserHistory}>
                     <Route path="home" component={Home}/>
-                    <Route path="help" component={Help}/>
-                    <Redirect from="*" to="home" />
+                    <Route path="help" component={() => (<Help problems={problemsData}/>)}/>
```
Comment (ShevchukJulia) [inline client/containers/App.js]: You definitely should use connect for this component to pass data from redux store into your Help component.
You should use examples from specification: 
https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options


In my opinion, for better decomposition you should create ProblemList component and wrap into react-redux connect component. And keep Home component simple and isolated from store.
For more information about architecture decomposition read: 
https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.odxxhoo1t

**Review by ShevchukJulia — CHANGES_REQUESTED** [review]

File: client/containers/problems_database.js (hunk 1 above)
Comment (Edgar000) [reply client/containers/problems_database.js]: i know it!

File: client/containers/App.js (hunk 2 above)
Comment (Edgar000) [reply client/containers/App.js]: w i p

**Review by ShevchukJulia — APPROVED** [review]

**Review by ShevchukJulia — APPROVED** [review]
ok)

File: client/components/Help/Help.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React, { Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import { Problem } from './../Problem/Problem';
+import './Help.scss';
+
+class Help extends Component {
+    componentDidMound() {
+        //fetch
```
Comment (OleksiiKukuruza) [inline client/components/Help/Help.js:8]: next time if you want to add todo - add comment in format: // TODO: some stuff

File: client/components/Problem/Problem.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes} from 'react';
+import './Problem.scss';
+
+export class Problem extends Component {
+    render() {
+        const problem = this.props.problem;
+        const isAnswered = (
```
Comment (OleksiiKukuruza) [inline client/components/Problem/Problem.js:7]: name this correctly please

File: client/components/Problem/Problem.scss [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  float: left;
+  margin-top: 3px;
+  padding: 5px 15px 5px 15px;
+  border-radius: 5px;
+  width: auto;
+  background: rgba(77, 0, 0, 1);
+  font-size: 20px;
+  color: white;
+
+  i {
```
Comment (OleksiiKukuruza) [inline client/components/Problem/Problem.scss:39]: As I said before - if you need style something - add class on that node. Avoid css selectors nesting

File: client/components/Problem/Problem.scss [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+.is-answered {
+  float: left;
+  clear: both;
+  border-radius: 5px;
+  height: 40px;
+  width: 40px;
+  background-color: white;
+
+  i {
```
Comment (OleksiiKukuruza) [inline client/components/Problem/Problem.scss:64]: same

File: client/components/Problem/Problem.scss [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+    margin: 4px 0 0 7px;
+    font-size: 30px;
+  }
+
+  .answered {
```
Comment (OleksiiKukuruza) [inline client/components/Problem/Problem.scss:69]: same

File: client/components/Problem/Problem.scss [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+    color: green;
+  }
+
+  .not-answered {
```
Comment (OleksiiKukuruza) [inline client/components/Problem/Problem.scss:73]: same

File: client/components/Problem/Problem.scss [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+.view-button {
+  float: left;
+  margin-left: 310px;
+  padding: 7px 5px 5px 10px;
+  border-radius: 5px;
+  background: rgba(200, 0, 0, 1);
+  font-size: 25px;
+  color: white;
+
+  i{
```
Comment (OleksiiKukuruza) [inline client/components/Problem/Problem.scss:98]: same

**Review by OleksiiKukuruza — APPROVED** [review]

**Pull request merged**